django/utils/tree.py

```diff
@@ -32,7 +32,7 @@ def create(cls, children=None, connector=None, negated=False):
         __init__() with a signature that conflicts with the one defined in
         Node.__init__().
         """
-        obj = Node(children, connector, negated)
+        obj = Node(children, connector or cls.default, negated)
```

Comment:
This was necessary to fix an issue that prevented `.create()` (previously `._new_instance()`) from being used by `Q()`. This is because `Node.__init__()` is called which falls back to `Node.default`, not `Q.default` as the default value for `connector`.